hydra_base/lib/scenario.py

```diff
@@ -122,7 +122,7 @@ def copy_data_from_scenario(resource_attrs, source_scenario_id, target_scenario_
 
     return target_resourcescenarios
 
-def get_scenario(scenario_id,**kwargs):
+def get_scenario(scenario_id, get_parent_data=False, **kwargs):
```

Comment:
Should `get_parent_data` default to `True` for scenarios with a parent?

Reply:
By making it false, the rational was to make retrieval of data not directly inside a scenario an explicit operation. This way i hope it will reduce misunderstanding of what the function is expected to return